<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="file_label_lights">
          <property name="text">
           <string>Light frames</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_lights"/>
        </item>
        <item>
         <widget class="QPushButton" name="choose_lights">
          <property name="text">
           <string>Choose folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="file_label_darks">
          <property name="text">
           <string>Dark frames</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_darks"/>
        </item>
        <item>
         <widget class="QPushButton" name="choose_darks">
          <property name="text">
           <string>Choose folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="file_label_flats">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Flat  frames</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_flats">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choose_flats">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="file_label_bias">
          <property name="text">
           <string>Bias frames</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_bias"/>
        </item>
        <item>
         <widget class="QPushButton" name="choose_bias">
          <property name="text">
           <string>Choose folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="alignMode">
        <property name="title">
         <string>Align mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QRadioButton" name="mode_triangles">
           <property name="text">
            <string>&amp;Triangle similarity star alignment (for DSO)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mode_ecc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>ECC a&amp;lignment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mode_center">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Center of &amp;gravity alignment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="stackMode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Stacking mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="stack_average">
           <property name="text">
            <string>Average</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="stack_median">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Median</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="stack_add">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="threshold_label">
          <property name="text">
           <string>Star detection threshold</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="threshold">
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="test_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QPushButton" name="stack_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Stack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="console_label">
        <property name="text">
         <string>Console output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="console">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_stars">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
